torchdata/datapipes/iter/load/online.py

```diff
@@ -119,14 +136,18 @@ def _get_response_from_google_drive(url: str, *, timeout: Optional[float]) -> Tu
         else:
             response = session.get(url, timeout=timeout, stream=True)
 
+        api_response, content = _extract_gdrive_api_response(response)
+        if api_response == "Quota exceeded":
+            raise RuntimeError(f"Google drive link {url} is currently unavailable, because the quota was exceeded.")
+
         if "content-disposition" not in response.headers:
             raise RuntimeError("Internal error: headers don't contain content-disposition.")
 
         filename = re.findall('filename="(.+)"', response.headers["content-disposition"])
         if filename is None:
             raise RuntimeError("Filename could not be autodetected")
 
-    return filename[0], StreamWrapper(response.raw)
+    return filename[0], StreamWrapper(content)
```

Comment:
Please keep it as `response.raw` to make the response in a streaming manner.

Reply:
I've replaced it back with `response.raw`. And I also made `_extract_gdrive_api_response` operate on the initial `response.text`, since it seems we expect the initial response to be an HTML page instead of the actual data anyways.